Mage.Sets/src/mage/cards/j/JyotiMoagAncient.java

```diff
@@ -0,0 +1,63 @@
+package mage.cards.j;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.common.BeginningOfCombatTriggeredAbility;
+import mage.abilities.common.EntersBattlefieldTriggeredAbility;
+import mage.abilities.dynamicvalue.common.CommanderCastFromCommandZoneValue;
+import mage.abilities.dynamicvalue.common.SourcePermanentPowerCount;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.continuous.BoostControlledEffect;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.constants.TargetController;
+import mage.game.permanent.token.ForestDryadToken;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.filter.common.FilterCreaturePermanent;
+
+/**
+ *
+ * @author grimreap124
+ */
+public final class JyotiMoagAncient extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("land creatures");
+
+    private static final SourcePermanentPowerCount xValue = new SourcePermanentPowerCount();
+
+    static {
+        filter.add(CardType.LAND.getPredicate());
+
+    }
+
+    public JyotiMoagAncient(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.CREATURE }, "{2}{G}{U}");
+
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.ELEMENTAL);
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(4);
+
+        // When Jyoti, Moag Ancient enters the battlefield, create a 1/1 green Forest Dryad land creature token
+        // for each time you've cast your commander from the command zone this game.
+        this.addAbility(new EntersBattlefieldTriggeredAbility(
```

Comment:
Last minor detail:
You can add a hint as second argument of `addAbility`.
In this case `CommanderCastFromCommandZoneValue.getHint()` is exactly the hint with the right info for the players to see on the card.

Reply:
When I was testing it already had the hint unless I missed

Reply:
Well your commander has one. But Jyoti might not be your commander.

Reply:
That probably would have been it. I'll update it